sip/sip.go

```diff
@@ -291,8 +292,10 @@ func EvaluateDispatchRule(rule *livekit.SIPDispatchRuleInfo, req *rpc.EvaluateSI
 	}
 	switch rule := rule.GetRule().GetRule().(type) {
 	case *livekit.SIPDispatchRule_DispatchRuleIndividual:
-		// TODO: Decide on the suffix. Do we need to escape specific characters?
-		room = rule.DispatchRuleIndividual.GetRoomPrefix() + from
+		// TODO: Do we need to escape specific characters in the number?
+		// TODO: Include actual SIP call ID in the room name?
+		ts := time.Now().UTC().Format(time.RFC3339)
+		room = fmt.Sprintf("%s_%s_%s", rule.DispatchRuleIndividual.GetRoomPrefix(), from, ts)
```

Comment:
any reason to not use [our GUID](https://github.com/livekit/protocol/blob/main/utils/id.go#L46) for this?

Reply:
Done :+1: